### PR TITLE
annotations to biigle scripts

### PR DESCRIPTION
**Pull request by marinmarcillat**
# Annotations_to_biigle

- Import pascalVOC annotations into Biigle.
- Automatic feature detection on Biigle images using yoloV5 models and annotations import back in Biigle via pascalVOC

### CODE REVIEW

File: annotations_to_biigle/annotations_to_biigle.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 # Biigle email and token
 email = 'mmarcill@ifremer.fr'
-token = 'jxSmYAj3uM7OOA1GIp5w2wCOyYMhb7vo'
```
Comment (mzur) [inline annotations_to_biigle/annotations_to_biigle.py]: Oops. Don't forget to revoke the token in BIIGLE. I'll review the PR soon!

Comment (marinmarcillat) [reply annotations_to_biigle/annotations_to_biigle.py]: Oops, thanks ! Done

File: annotations_to_biigle/export_to_biigle.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        elif shape == "Rectangle":
+            points = [int(row["xmin"]), int(row["ymin"]), int(row["xmax"]), int(row["ymin"]), int(row["xmax"]),
+                      int(row["ymax"]), int(row["xmin"]), int(row["ymax"])]
+
+        for label in label_idx:
+            if label[0] == row["name"]:
+                post_data['label_id'] = label[1]  # Get label biigle id
+        post_data['points'] = points
+        post_data['confidence'] = float(row["confidence"])
+        p = api.post('images/{}/annotations'.format(image_id), json=post_data)  # Post payload to biigle
```
Comment (mzur) [inline annotations_to_biigle/export_to_biigle.py:110]: If you have lots of annotations you can use the [endpoint](https://biigle.de/doc/api/index.html#api-ImageAnnotations-BulkStoreImageAnnotations) to create up to 100 annotations in a single request. This is much faster. I let you decide if you want to implement this or rather leave it like that.

File: README.md [hunk 3]
```diff
@@ -6,16 +6,16 @@ A collection of community-maintained scripts that use the BIIGLE API.
...
+| [`convert-laser-points`](convert-laser-points)   | Create annotations from the positions of a laser point detection.                                        |
+| [`grab-largo-patches`](grab-largo-patches)       | Download the Largo patches of annotations with a specific label.                                         |
+| [`regular-sampling-grid`](regular-sampling-grid) | Create annotations that form a regular grid for all images of a volume.                                  |
+| [`create-volume`](create-volume)                 | Create a new volume based on files found in a local directory.                                           |
+| [`biigle_to_coco`](biigle_to_coco)               | Create a JSON report in the standard COCO format from the BIIGLE CSV image annotation report.            |
+| [`check_coordinates`](check_coordinates)         | Parses a CSV image annotation report file and adjusts all annotation coordinates to be inside the image. |
+| [`annotations_to_biigle`](annotations_to_biigle) | Import pascalVOC annotations to Biigle, automatic detection and import using yoloV5 models.              |
```
Comment (mzur) [inline README.md:18]: Please call it "BIIGLE".

**Review by mzur — CHANGES_REQUESTED** [review]
Sorry for the long wait! I've just added a few minor comments. But before I can merge could you please resolve the conflict in the `README.md`?

**Review by mzur — APPROVED** [review]
Thanks!

File: annotations_to_biigle/export_to_biigle.py (hunk 2 above)
Comment (marinmarcillat) [reply annotations_to_biigle/export_to_biigle.py:110]: Thanks for the idea ! I just implemented it, its on my fork: https://github.com/marinmarcillat/community-scripts

Should I do another pull request for this ?

Sorry, I'm not yet very familiar with git and every commands, it's my first pull request

Comment (mzur) [reply annotations_to_biigle/export_to_biigle.py:110]: Sorry then I was too quick with the merge. Yes you can just send another pull request!

**Pull request merged**